waiverdb/templates/permissions.html

```diff
@@ -14,28 +14,25 @@
                         <th>Name</th>
                         <th>Maintainers</th>
                         <th>Testcases</th>
-                        <th>Users/Groups</th>
+                        <th>Groups</th>
+                        <th>Users</th>
+                        <th>Description</th>
```

Comment:
I found that only few rules actually use description. That's why I threw it away.

Reply:
Ideally, description should be included in other permissions too. Otherwise we don't have any context about the waivers or usage.